package.json

```diff
@@ -35,7 +35,8 @@
     "test": "ava",
     "universal": "napi universal",
     "version": "napi version",
-    "docs": "yarn build && npx typedoc --plugin typedoc-plugin-markdown index.d.ts"
+    "docs-html": "yarn build && npx typedoc index.d.ts",
+    "docs-md": "yarn build && npx typedoc --plugin typedoc-plugin-markdown index.d.ts"
```

Comment:
The only change that isn't auto-generated

Reply:
Going forward, I definitely think markdown files that are baked into the Envio docs would be best. And even here having a separate branch for docks would also be cleaner so that pull requests don't get cluttered with all of these changes. I just don't want to rock the boat or get distracted by this at this very moment.